kfdefs/base/monitoring/blackbox-exporter/blackbox-prober-config.yaml

```diff
@@ -0,0 +1,12 @@
+modules:
+  http_2xx:
+    prober: http
+    timeout: 30s
+    http:
+      valid_status_codes: [200, 201, 202, 203, 204, 205, 206, 207, 208, 403]
```

Comment:
I feel like 403 isn't necessarily a good response code. Since our probe should always be able to authenticate to the service, it _failing_ to authenticate means something changed. I might be overthinking it here though :-)

Reply:
Can we remove 403 from here?